src/Relation/Binary/PropositionalEquality.agda

```diff
@@ -104,15 +104,17 @@ module _ (_≟_ : DecidableEquality A) {x y : A} where
   ≢-≟-identity = dec-no (x ≟ y)
 
 
+------------------------------------------------------------------------
+-- Inspect
 
+-- Inspect can be used when you want to pattern match on the result r
```

Comment:
'result' or 'value'?
(I realise this is the previous text, but I wonder if this is more precise usage?)